packages/block-editor/src/components/roving-tab-index/README.md

```diff
@@ -0,0 +1,47 @@
+RovingTabIndex
+==============
+
+`RovingTabIndex` exports two separate components `<RovingTabIndex.Container />` and `<RovingTabIndex.Item />` that help to implement a roving tab index.
+
+A roving tab index is helpful for a UI component that contains multiple focusable elements, it reduces the number of tab stops in such an element to a single tab stop. Some more information is available here:
+
+- [WAI Aria Authoring Practices - Roving Tab Index](https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_roving_tabindex)
+
+Some patterns that implement a roving tab index are:
+
+- [Layout Grid](https://www.w3.org/TR/wai-aria-practices/examples/grid/LayoutGrids.html)
+- [Editor Menu Bar](https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-2/menubar-2.html)
+- [Navigation Menu Bar](https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html)
+- [Radio Group](https://www.w3.org/TR/wai-aria-practices/examples/radio/radio-1/radio-1.html)
+- [Toolbar](https://www.w3.org/TR/wai-aria-practices/examples/toolbar/toolbar.html)
```

Comment:
Noting that for the fully accessible version of Toolbar in `@wordpress/components` we use [Rover](
https://reakit.io/docs/rover/) from Reakit. 

Based on the Reakit's documentation only, I don't think it supports grids though.

Reply:
Is that in an open PR? I couldn't find it in the codebase.

As you say it doesn't seem to support grids. The RovingTabIndex components in this PR only handle focus management, and they're intended to be composed with a separate component for keyboard navigation, so that's the main difference.

Reply:
It's part of the codebase. It still isn't integrated with the block editor because of the required refactor to the SlotFill implementation.

You can test it in Storybook:
https://wordpress.github.io/gutenberg/?path=/story/components-toolbar--default